held_suarez/held_suarez.meta

```diff
@@ -0,0 +1,85 @@
+[ccpp-arg-table]
+  name  = held_suarez_1994_init
+  type  = scheme
+[ cappa_in ]
+  standard_name =  ratio_of_dry_air_to_water_vapor_gas_constants
+  units = 1
+  type = real | kind = kind_phys
+  dimensions = ()
+  intent = in
+[ cpair_in ]
+  standard_name = specific_heat_of_dry_air_at_constant_pressure
+  units = J kg-1 K-1
+  type = real | kind = kind_phys
+  dimensions = ()
+  intent = in
+[ psurf_ref_in ]
+  standard_name = reference_pressure_at_sea_level
+  units = Pa
+  type = real | kind = kind_phys
+  dimensions = ()
+  intent = in
+[ pref_mid_norm_in ]
+  standard_name = reference_pressure
+  units = Pa
+  type = real | kind = kind_phys
+  dimensions = (vertical_layer_dimension)
+  intent = in
+
+[ccpp-arg-table]
+  name  = held_suarez_1994_run
+  type  = scheme
+[ ncol ]
+  standard_name = horizontal_loop_extent
+  units = count
+  type = integer
+  dimensions = ()
+  intent = in
+[ clat ]
+  standard_name = latitude
+  units = radians
+  type = real | kind = kind_phys
+  dimensions = (horizontal_loop_extent)
+  intent = in
+[ pmid ]
+  standard_name = air_pressure
+  units = Pa
+  type = real | kind = kind_phys
+  dimensions = (horizontal_loop_extent,vertical_layer_dimension)
+  intent = in
+[ u ]
+  standard_name = eastward_wind
+  units = m s-1
+  type = real | kind = kind_phys
+  dimensions = (horizontal_loop_extent,vertical_layer_dimension)
+  intent = in
+[ v ]
+  standard_name = northward_wind
+  units = m s-1
+  type = real | kind = kind_phys
+  dimensions = (horizontal_loop_extent,vertical_layer_dimension)
+  intent = in
+[ t ]
+  standard_name = temperature
+  units = K
+  type = real | kind = kind_phys
+  dimensions = (horizontal_loop_extent,vertical_layer_dimension)
+  intent = in
+[ du ]
+  standard_name = tendency_of_eastward_wind
+  units = m s-2
+  type = real | kind = kind_phys
+  dimensions = (horizontal_loop_extent,vertical_layer_dimension)
+  intent = out
+[ dv ]
+  standard_name = tendency_of_northward_wind
+  units = m s-2
+  type = real | kind = kind_phys
+  dimensions = (horizontal_loop_extent,vertical_layer_dimension)
+  intent = out
+[ s ]
+  standard_name = tendency_of_temperature
+  units = K s-1
```

Comment:
It might make more sense for this variable to be labeled `ds`, instead of `s`, to clarify that it is in fact a tendency, not a state variable.

Also, the standard name here should say `tendency_of_dry_static_energy`, or something equivalent, as the temperature is being multiplied by `cpair` in the actual fortran routine (which converts it to static energy).

Finally, the units here should be `J kg-1 s-1`.